src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}
+

```

Comment:
redundant blank lines
```suggestion
    public abstract void doWork();

    public abstract void stopWork();
}
```